saltstack/base/salt/postgresql/init.sls

```diff
@@ -146,3 +146,7 @@ configure-listen-address:
 {% if pillar['subtype'] != 'Docker' %}
       - service: start-postgresql
 {% endif %}
+
+postgres:
+  user.present:
+    - shell: /usr/sbin/nologin
```

Comment:
could you give a meaningful state name instead of using the user as the state name? eg:
```
set-postgres-login-shell:
  user.present:
    - name: postgres
    - ...
```